install_files/ansible-base/roles/backup_app/tasks/backup.yml

```diff
@@ -1,18 +1,15 @@
 ---
-  # TODO the version is hardcoded into the collect.py script name and ansible
-  # task. This should be changed in the ansible task to use a group_var with the
-  # version or to read it from the securedrop_code version.py
-- name: copy the versions collect.py script to the target server /tmp directory
+- name: copy the version\'s collect.py script to the app server /tmp directory
   copy:
-    src: 0.3_collect.py
+    src: "{{ backup_script_version }}_collect_app.py"
```

Comment:
@conorsch Can you have subdirectories in an Ansible role's `files/` directory? I think it's a little neater to use subdirectories instead of interpolated filenames in a case like this, e.g.

```
roles/backup/files/0.3/collect_app.py
roles/backup/files/0.3/restore_app.py
...
```